assets/js/vue/components/research-group/PublicationsTab.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+    <b-card-group>
+        <PublicationRow :publicationRow="publication" v-for="publication in getPublications()" v-bind:key="publication.id"/>
+    </b-card-group>
+</template>
+
+<script>
+    import PublicationRow from "./PublicationRow";
+    export default {
+        name: "PublicationsTab",
+        components: { PublicationRow },
+        props: {
+            datasets: {}
+        },
+        methods: {
+            getPublications: function () {
+                let publications = [];
+                let publicationId = [];
+                this.datasets.forEach(dataset => {
+                    dataset.publications.forEach(publication => {
```

Comment:
Not a fan of using nested FOR loops, couldn't find a better way. Please let me know if there is an efficient way to do this

Reply:
Maybe you could use `array_unique` ?

Reply:
The overhead of the inefficiency of the nested loops is negligible with the relative small numbers we're ever likely to deal with, and the inner loop isn't doing that much anyway. I'm ok with it as is, but Mike's suggestion would be fine too.

Reply:
Isn't array_unique a Php function?